webserver/app/celery_app.py

```
@@ -0,0 +1,41 @@
+    inspector = celery.control.inspect()
+    stats = inspector.stats()
+    
+    if not stats:
+        return True
+        
+    active_tasks = sum(len(worker['active']) for worker in stats.values())
+    reserved_tasks = sum(len(worker['reserved']) for worker in stats.values())
+    
+    return active_tasks == 0 and reserved_tasks == 0
+
+def shutdown_gcp_instance():
```

Ruff (0.11.9)

Missing return type annotation for public function `shutdown_gcp_instance`

Add return type annotation: `None`

(ANN201)